### ISSUE TITLE
Download doesn't start: "waiting for it to come online..." 

### ISSUE THREAD

**Issue by hybam-dev** [CS-SI/eodag-sentinelsat#31]
Hi, 
After having installed and configured the credentials properly, eodag can find the products in scihub, but it always freeze the "waiting for it to come online" message. 
I have already raised the logging to level 3 to check for other insights, but couldn't find a fix.
Here is the output:
![image](https://user-images.githubusercontent.com/59015615/130614348-9515856c-8d08-43f5-86b6-1f7bb21187ed.png)

Any ideas?
Thanks,
Mauricio

**Comment by sbrunato** [comment]
Hi,
regarding your logs, you might not run the latest version. `EODAG-sentinelsat` was recently updated to `v0.4.0`, to help handling LTA products download.

`download()` or `download_all()` will trigger a product request, then loop to retry downloading until products are available. By default eodag will retry downloading every 2 minutes for 20 minutes. These parameters can be tuned using `wait` and `timeout` (both in minutes):
```py
paths = dag.download_all(search_result, wait=0.5, timeout=120)
# will try downloading search_result every 30" for 2h
```
For more details, see [EODAG API Reference: `EODataAccessGateway.download`](https://eodag.readthedocs.io/en/stable/api_reference/core.html#eodag.api.core.EODataAccessGateway.download) and [eodag-sentinelsat v0.4.0  release notes](https://github.com/CS-SI/eodag-sentinelsat/releases/tag/v0.4.0)

**Comment by sbrunato** [comment]
Thanks for reporting this issue. Sentinelsat just released to `v1.1.0` and we'll check if `eodag-sentinelsat` needs to be updated to match its updated API.

**Comment by sbrunato** [comment]
>  it always freeze the "waiting for it to come online" message.

In fact it is not frozen, but the download retry progress  is not logged by sentinelsat. There is only this message at the beginning, then another one when it succeeds or fails.  See [sentinelsat/download.py#L617-L645](https://github.com/sentinelsat/sentinelsat/blob/a551d071f9c5faae09603ec4a3ef9dc3dd3ef833/sentinelsat/download.py#L617-L645)

In order to homogenize with other eodag providers, the retry mechanism will be more handled on eodag side than it is for the moment.
